var/spack/repos/builtin/packages/py-libensemble/package.py

```diff
@@ -45,6 +46,7 @@ class PyLibensemble(PythonPackage):
     variant("deap", default=False, description="Install with DEAP")
     variant("tasmanian", default=False, description="Install with tasmanian")
     variant("pyyaml", default=False, description="Install with pyyaml")
+    variant("pytoml", default=False, description="Install with pytoml")
 
```

Comment:
All of the following dependencies are required and shouldn't have variants, at least in the latest version:
```python
    install_requires=["numpy", "psutil", "setuptools", "pydantic", "tomli", "pyyaml"],   
```
The only variants I see listed explicitly in the package are +extras and +docs.

Reply:
Fixed